test/Microsoft.ML.Core.Tests/UnitTests/DataTypes.cs

```diff
@@ -0,0 +1,322 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.Conversion;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Runtime.RunTests
+{
+    public class DataTypesTest : TestDataViewBase
+    {
+        public DataTypesTest(ITestOutputHelper helper)
+           : base(helper)
+        {
+        }
+
+        private readonly static Conversions _conv = Conversions.Instance;
+
+        [Fact]
+        public void TXToI1()
```

Comment:
(nit) can we use standard .NET type names here?  `TXToSByte`?

Reply:
repeated below for other "Ix" names.

---
In reply to: [213516290](https://github.com/dotnet/machinelearning/pull/683#discussion_r213516290) [](ancestors = 213516290)